src/examples/orb-dev-workflows.yml

```diff
@@ -19,6 +19,13 @@ usage:
     # publish manually once before this worklow can succeed)
     your-orb: your-namespace/your-orb@<<pipeline.parameters.dev-orb-version>>
 
+  orb_prep_jobs: &orb_prep_jobs
+    [
+      orb-tools/lint,
+      orb-tools/shellcheck,
+      orb-tools/pack
+    ]
```

Comment:
Is there a reason you switched to JSON style here rather than traditional YAML style?

Reply:
In YAML Anchors and Aliases I have just seen this pattern most often used. I may have done it incorrectly in the past but I was not able to find the working yaml syntax for this situation, but I could give it another go.

Reply:
Typically `[ something, something else]` is the JSON style alternative for:

```
- something
- something else
```

in YAML. Maybe the normal YAML syntax doesn't work here because of the alias above?